Enhance button

yousseefben opened the pull request.

Commit:
support fullwidth button

Commit:
ehance input

Commit:
enhance dropdown

Commit:
fix style view

MoritzGruber commented on src/components/button/styles/SButton.tsx:
```diff
@@ -356,7 +356,12 @@ const Bronze = css<{ styleType: IButtonType } & IButtonProps>`
     align-items: center;
   }
-  ${({ styleOverwrite }) => styleOverwrite ?? ''};
+  ${(props) =>
```
@yousseefben  in my opinion we can just pass this 100% with the style overwrite - that is what the style overwrite is ment to be fore. Otherwise if we add an extra prop for every style thing it will get messy quickly

MoritzGruber replied:
so please remove the fullwith thing and use styleOverwrite in web-app

yousseefben replied:
okay i'll do, just i found that is cleaner lik eother libraries because we use it a lot, i'll push it now

MoritzGruber commented on src/components/dropdown-card/styles/SDropdownCardBody.tsx:
```diff
@@ -1,8 +1,8 @@
 
 const Bronze = css`
-padding: 20px;
+  padding: ${(props) => `${props.padding || 20}px`};
```
@yousseefben  also here i would not use a padding property - it cleaner if we create a styleOverwrite like in the button and put the padding we need there in web-app

yousseefben replied:
okay, i don't need the padding anymore, so i'll rollback

MoritzGruber commented on src/components/line-input/styles/SLineInputLabel.ts:
```diff
@@ -5,9 +5,14 @@ import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../consta
-  opacity: 0;
-  transition: opacity 200ms, color 200ms;
+  ${(props) =>
+    !props.isLabelVisible &&
```
@yousseefben  can you share the use-case for this?

MoritzGruber replied:
because of the branch name there is no more link to linear so its hard for me to review if we need this

yousseefben replied:
ah i see, it's on the dopdownsaveview, the label is by default hidden, so on figma we should see the label here the [link](https://www.figma.com/file/XrUkpZwlWi2swxzvpte03J/Screens-%26-Flows?node-id=258%3A44200) to figma

Commit:
rollback

Commit:
rollback fullwidth

Commit:
chore: remove labelVisible thing since there was

chore: remove labelVisible thing since there was no need for this after talk with design
